examples/applications/TeleSculptorCLI.py

```diff
@@ -0,0 +1,178 @@
+import argparse
+import os
+
+START_STEP = 1
+END_STEP = 5
+
+TRACK_FILE = "results/tracks.txt"
+CAMERA_DIR = "results/krtd"
+LANDMARKS_FILE = "results/landmarks.ply"
+DEPTH_DIR = "results/depth"
+VOLUME_FILE = "results/volume.vti"
+MESH_FILE = "results/mesh.vtp"
+TEXTURED_MESH_FILE = "results/textured_mesh.vtp"
+
+
+def run_and_return(cmd_string, dry_run=False):
+    print("\nAbout to run `{}`".format(cmd_string))
+    if not dry_run:
+        ret_val = os.system(cmd_string)
+        return ret_val
+    return 0
+
+
+parser = argparse.ArgumentParser(
+    """
+        Before running, make sure to `cd` to the build directory and `source`
+        the appropriate setup_KWIVER.<extension> script for your platform
+
+        Providing just the required arguments (kwiver binary and video) will run
+        the process end-to-end on that video with default parameters. Or you
+        can run a subset of the steps in the TeleSculptor pipeline
+        (1) Track features
+            Requires video-file
+        (2) Initialize cameras
+            Requires video-file, track-file
+        (3) Estimate depths per view
+            Requires video-file, camera-dir
+        (4) Fuse depths into a mesh
+            Requires camera-dir, depth-dir
+        (5) Colorize mesh based on source imagery
+            Requires video-file, camera-dir, mesh-file\n
+        """,
+    formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+
+# Required positiononals
+parser.add_argument(
+    "kwiver", help="Path to the kwiver binary")
+parser.add_argument(
+    "video",
+    help="The source video to run on. Can also be a text file containing a list of newline-seperated image paths")
```

Comment:
I'm not sure what code style standards we are using for Python in KWIVER (PEP8?) but we should break long lines

Reply:
Yes, it is PEP8. There is no explicit guidance on how to handle long string, and `autopep8` just ignores them. What I did is my preference, but I'm open to other options.